code/datums/status_effects/buffs.dm

```diff
@@ -129,12 +129,12 @@
 /datum/status_effect/blooddrunk/on_remove()
 	if(ishuman(owner))
 		var/mob/living/carbon/human/H = owner
-		H.physiology.brute_mod *= 10
-		H.physiology.burn_mod *= 10
-		H.physiology.tox_mod *= 10
-		H.physiology.oxy_mod *= 10
-		H.physiology.clone_mod *= 10
-		H.physiology.stamina_mod *= 10
+		H.physiology.brute_mod *= 4
+		H.physiology.burn_mod *= 4
```

Comment:
To prevent this happening the future could you make this a variable on the status effect and multiply/divide by it